ypkg2/main.py

```diff
@@ -380,6 +416,7 @@ def build_package(filename, outputDir, buildDir=None):
                 end_time = timer()
                 console_ui.emit_success("Build", "{} successful ({})".
                                         format(step, timedelta(seconds=end_time-start_time)))
+                post_execute_step(context, r_step, step, work_dir)
```

Comment:
It looks like this is run on every step of the build. I think it would make more sense to check if we're on the "install" step before running it here instead of inside the `post_execute_step()` function; it also slightly simplifies that function.